src/popup/pages/Dashboard.vue

```diff
@@ -22,6 +22,8 @@
         />
       </div>
 
+      <a @click="openMultisigTransactionProposalModal()">Transaction proposal</a>
```

Comment:
this is not a link, more semantic is option is use button

Reply:
Transaction proposal -> to translation

Reply:
This is temporary thing. I added TODO comment.